Content.Server/StationEvents/Components/VentCrittersRuleComponent.cs

```diff
@@ -11,6 +11,6 @@ public sealed class VentCrittersRuleComponent : Component
         "MobMouse",
         "MobMouse1",
         "MobMouse2",
-        "MobMothroach"
+        "MobMothroach" // nyano change: Mothroach
```

Comment:
This entire block is obsolete on a yet to be pushed upstream merger branch.

Reply:
ree.